AcademyApp/Extensions/UIColorExtensions.swift

```diff
@@ -15,6 +15,10 @@ extension UIColor {
 
     // MARK: - Static properities
 
+    static var lectureNameTextColor: UIColor {
```

Comment:
Its necessary to define own color which is white? 
is it good to have for each lable own defined color?
what if lectureNameText has various colors in various screens?

Reply:
Removed lectureNameTextColor from UIColor extensions.